src/components/ChatPanel.tsx

```diff
@@ -107,15 +108,25 @@ export function ChatPanel(props: ChatPanelProps) {
       <div className={cssClasses.container}>
         {header}
         <div className={cssClasses.messagesScrollContainer}>
-          <div ref={messagesRef} className={cssClasses.messagesContainer}>
-            {messages.map((message, index) => (
-              <MessageBubble
-                {...props}
-                customCssClasses={cssClasses.messageBubbleCssClasses}
-                key={index}
-                message={message}
-              />
-            ))}
+          <div ref={messagesContainer} className={cssClasses.messagesContainer}>
+            <div ref={messagesRef}>
```

Comment:
a small concern with adding a new div here is that certain styling changes to `messagesContainer` will no longer reflect all message bubbles as expected -- since the children is now a big div of all previous message bubble + last message bubble, instead of a list.

Reply:
makes sense! I think i've found a workaround, let me know what you think